content/themes/biojs-theme/partials/tip-section.hbs

```diff
@@ -2,11 +2,11 @@
 <div class="container">
 <div class="row">
 <div class="col s12 m4">
-  <div class="card biojs-dark-grey biojs-white-text">
-    <div class="card-content white-text">
```

Comment:
@ayusharma Yes, good one! Can you also check if you need to apply another helper to fix the bug?